gradoop-flink/src/main/java/org/gradoop/flink/model/impl/functions/epgm/EdgeFromIds.java

```diff
@@ -28,6 +28,7 @@
 
 /**
  * Initializes an {@link Vertex} from a given {@link GradoopId} triple.
+ * The triple consists of edge id, source vertex id and target vertex id.
  *
  * (edgeId, sourceId, targetId) -> edge
```

Comment:
This might not be the scope of this PR, but please put a `{@code }` around this or remove it (the comment above should be sufficient)